package.json

```diff
@@ -1,15 +1,37 @@
 {
-  "name": "code-exercise",
-  "version": "1.0.0",
-  "description": "This is a shopping cart coding exercise",
-  "main": "index.js",
-  "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "canned -p 5000 ./server"
-  },
-  "author": "",
-  "license": "MIT",
-  "dependencies": {
-    "canned": "^0.3.12"
-  }
+    "name": "code-exercise",
+    "version": "1.0.0",
+    "description": "This is a shopping cart coding exercise",
+    "main": "app.js",
+    "scripts": {
+        "test": "echo \"Error: no test specified\" && exit 1",
+        "gulp": "gulp",
+        "build": "babel ./app.js --out-file myapp.js && babel ./resources/javascripts --out-dir ./resources/js ",
```

Comment:
also have a 'build' folder to have your babelified code

Reply:
ok